python/test/test_tests.py

```diff
@@ -1,3 +1,29 @@
+import opendp.prelude as dp
+
+
 def test_number_of_tests_found(request):
     tests_found = len(request.session.items)
     assert tests_found >= 304
+
+
+def test_barely():
```

Comment:
If we're not really testing them, we shouldn't take credit! Can these be distributed into the corresponding tests? Is it possible to tweak any of those tests to get full coverage?

Reply:
Where I feel confident about the behavior I'll try to add stronger assertions and move to a more appropriate file. I'm not sure I'll be able to do that for all.

In the background, there is the question of what "test coverage" means -- **Real** 100% coverage is something to strive for ... but what we've been doing is requiring "100%", but actually having exceptions scattered all over the place. Rather than sweeping problems under the rug, I was thinking of this as a centralized TODO list, and if new untested generated code is added, CI would fail unless we have a TODO test here, or a real test elsewhere.

There are projects that have a "100% test coverage" badge, but I certainly wouldn't advertise that for opendp.